server/cells.go

```diff
@@ -71,6 +70,12 @@ func newCellResponses(dID chronograf.DashboardID, dcells []chronograf.DashboardC
 // have the correct axes specified
 func ValidDashboardCellRequest(c *chronograf.DashboardCell) error {
```

Comment:
This function should verify that `c` is non-nil and return an error if it is to prevent possible panic.